tests/test_exmol.py

```diff
@@ -87,6 +87,15 @@ def model(s, se):
     exmol.sample_space("CCCC", model, batched=False, quiet=True)
 
 
+def test_sample_f():
+    def model(s):
+        return int("N" in s)
+
+    # try both SMILES and SELFIES
+    exmol.sample_space("CCCC", model, batched=False, use_selfies=True)
+    exmol.sample_space("CCCC", model, batched=False, use_selfies=False)
+
+
 def test_sample_preset():
     def model(s, se):
         return int("N" in s)

```

Comment:
Can we change all tests to `model(s)`?

Reply:
We could, but would like to keep functionality the same for existing code (e.g., paper notebooks) or other's code. For sure, all new tests can use this syntax though